### PR TITLE
[Bug #158140829] Restrict room resource addition to  admin users

### PR DESCRIPTION
**Pull request by dkam26**
#### What does this PR do?
- Restrict room resource addition to only authenticated admin users only

#### Description of the task to be completed**

- Room resources can currently be added by any users.Restrict addition of room resources to only authenticated admin users.

#### What are the relevant pivotal tracker stories?

- #158140829

#### Screenshots

image

### CODE REVIEW

**Comment by PatrickCmd** [comment]
@dkam26 Please correct the formatting of words on your PR.

File: tests/test_room_resource/test_create_room_resource.py [hunk 1]
```diff
@@ -4,6 +4,10 @@
     resource_mutation_0_value_room_id_query, error_0_value_room_id,
     resource_mutation_empty_name_string_query, error_empty_name_string
 )
```
Comment (PatrickCmd) [inline tests/test_room_resource/test_create_room_resource.py]: @dkam26 I think some of these imports are not required and used. So they should be removed and then flake8 will pass.

**Comment by PatrickCmd** [comment]
@dkam26 Your PR is behind develop, please rebase it accordingly with develop.

File: tests/test_room_resource/test_create_room_resource.py (hunk 1 above)
Comment (dkam26) [reply tests/test_room_resource/test_create_room_resource.py]: @PatrickCmd ,Thanks for the feedback.I have made the necessary changes

**Pull request merged**